README.md

```diff
@@ -1,34 +1,40 @@
 Takeaway Challenge
 ==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
 
- ```
+    \\\\\\\|///////
```

Comment:
Maybe consider having: Domain model // IRB. An IRB would be helpful to see how this interacts i.e. in mine its like a new user is made and then does x y z so you can (in theory) follow the flow.

Reply:
Yes I need to start adding these to my readme.